ykman/scancodes/__init__.py

```diff
@@ -27,16 +27,17 @@
 
 from __future__ import absolute_import
 from enum import Enum
-from . import us, de, fr, it, modhex, norman
+from . import us, de, fr, it, modhex, norman, bepo
 
 
 class KEYBOARD_LAYOUT(Enum):
     MODHEX = modhex.scancodes
     US = us.scancodes
     DE = de.scancodes
-    NORMAN = norman.scancodes
     FR = fr.scancodes
     IT = it.scancodes
+    BEPO = bepo.scancodes
+    NORMAN = norman.scancodes
```

Comment:
Is it time to sort these? :smile:

Reply:
I did sort them from default to most obscure..

Reply:
Ah, I see. :)